app/src/main/res/values/colors.xml

```diff
@@ -134,4 +134,6 @@
 	<color name="color_cyan7">#1098ad</color>   <!-- not_used -->
 	<color name="color_cyan8">#0c8599</color>   <!-- not_used -->
 	<color name="color_cyan9">#0b7285</color>
+	<color name="disabled_test_text">#cccccc</color>
```

Comment:
This should be a color from our color scheme.

I suggest using #ced4da (gray4)